templates/20_workernode.config.erb

```diff
@@ -42,6 +42,10 @@ STARTD_CRON_WN_HEALTHCHECK_KILL = true
 
 ## When is this node willing to run jobs?
 START = (NODE_IS_HEALTHY =?= True) && (StartJobs =?= True)
+<% if ! @start_always_users.empty? then -%>
+## Some users can also run jobs if the node is not healthy / startjobs is false.
+START = ($(START)) || ( <%= @start_always_users.map { |s| "TARGET.Owner =?= \"#{s}\"" }.join(' || ') %> )
+<% end -%>
```

Comment:
Since this is an if-else, there should not be an `<% end -%>`, right?

```erb
<% if @condition -%>
  bar = 42
<% else -%> 
   bar = 0
<% end -%>
```

Reply:
Indeed. I wonder why this worked in the first place without throwing any error... Updated version incoming!